ops/_main.py

```diff
@@ -488,6 +496,7 @@ def _make_framework(self, dispatcher: _Dispatcher):
 
     def _emit(self):
         """Emit the event on the charm."""
+        ops.charm._setup_tracing(self.charm)
```

Comment:
I find it confusing that there's `ops.charm._setup_tracing` and `ops._tracing.setup_tracing`, which do quite separate things. Could one be renamed? Maybe this one, to be more focused on "emit the event", which is all it's doing at the moment, or "request destination" since that's what it's wanting?